spec/serializers/framework_assessment_serializer_spec.rb

```diff
@@ -5,7 +5,8 @@
 RSpec.describe FrameworkAssessmentSerializer do
   subject(:serializer) { described_class.new(assessment, includes) }
 
-  let(:move) { create(:move) }
+  let(:location) { create(:location) }
```

Comment:
why do we need define a specific location here?

Reply:
It's to circumvent a validation rule when creating a youth risk assessment - but actually I don't need that now as I removed the specs for the `amended_at` attribute here. Good spot, I'll clean this up.